docs/how-monty-works/faq-monty.md

```diff
@@ -81,8 +111,51 @@ While this remains an active area of research in deep-reinforcement learning ([H
+
+## Is the Thousand Brains Theory Related to "Symbolic" Artificial Intelligence?
+
+Concepts from symbolic AI have some similarities to the Thousand Brains Theory, including the importance of discrete entities, and mapping how these are structurally related to one another. However, we believe that it is important that representations are grounded in a sensorimotor model of the world, whereas symbolic approaches typically begin at high levels of abstraction.
+
+However, the approach we are adopting contrasts to some "neuro-symbolic" approaches that have been proposed. In particular, we are not attempting to embed entangled, object-impoverished deep-learning representations within abstract, symbolic spaces. Rather, we believe that object-centric representations using reference frames should be the representational substrate from the lowest-level of representations (vision, touch) all the way up to to abstract concepts (languages, societies, mathematics, etc.). Such a commonality in representation is consistent with the re-use of the same neural hardware (the cortical column) through the human neocortex, from sensory regions to higher-level, "cognitive" regions.
+
+# Applications of Monty
+
+## Can Monty be used for a scientific problem, like mapping the genetic sequence of a protein to its 3D structure?
+
+This depends a lot on how data is available. Given a static dataset of genetic sequences and their mapping onto the 3D structures of their proteins, Monty is not going to work well, while this is where a function-approximation algorithm like deep-learning can excel.
+
+Where Monty would eventually shine is when it is able to control experimental devices that allow it to further probe information about the world, i.e. a sensorimotor environment. For exampple, Monty might have some hypotheses about a structure, and want to test these through various experiments, probing the space in which it is uncertain. We embed representations in 3D coordinates that can take on any kind of graph structure necessary which is <= 3D space (strings, graphs defined by edges, or 3D point-clouds), so in theory these kinds of entities can all be represented.
+
+How Monty would *learn* to generalize a mapping between these levels of representations remains outstanding, and relates to how Monty can learn a mapping between different spaces (e.g. meeting a new family, and mapping these people onto an abstract family-tree structure). We are still figuring out exactly how this would work in a simpler case like the family-tree. In the protein case, the rules are much more complex, and so learning this is definitely not something that Monty can do now. However, just like a human scientist, the ultimate aim would be for Monty to learn to do this mapping based on a causal understanding of the world, informed by the above mentioned experiments, and in contrast to an end-to-end black-box function approximator.
```

Comment:
Do you think it could be nice to add the figure you had in the symposium slides about how ANNs are great as general function approximations, calculators for math and Monty for sensorimotor learning?
It would also be nice to add a cross link to this page: https://thousandbrainsproject.readme.io/docs/application-criteria

Reply:
Good idea about the cross-link, I'm worried the image might be confusing because the argument here is basically that, under the right conditions, Monty could *also* eventually do protein prediction?